lms/templates/index.html

```diff
@@ -3,7 +3,7 @@
 <%namespace name='static' file='static_content.html'/>
```

Comment:
@jcdyer, this change is already present in hawthorn. So it can be removed from the theme.